src/modules/application/controllers/controllerEdit.js

```diff
@@ -0,0 +1,68 @@
+import { verifyToken } from "../../../utilities/jsonWebToken.js";
+import { databasePrisma } from "../../../prismaClient.js";
+
+export const handleEdit = async function (req) {
+  const { id: applicationId } = req.params;
+  const { applicant, listing, coverLetter, company } = req.body;
```

Comment:
Technically the only thing we need in the request body is the cover letter, the rest shouldn't be changed. The application Id is taken from the params, and the users identity can be matched up to the application through their authorization token.

Reply:
Updated `controllerEdit.js`
Updated QA PUT endpoint accordingly